Source/textdat.cpp

```diff
@@ -696,6 +696,44 @@ const TextDataStruct alltext[] = {
 	    1, 5, TSFX_BMAID30 },
 	{ "I really don't know much about the priest Tremain. He never visits the tavern, preferring to keep to the company of Pepin and Cain. Perhaps it is because they, too, have more scholarly pursuits."
 	    1, 5, TSFX_BMAID38 },
+	{ "Roof and Inaius are missing? Light protect us! Is there no place that is safe? Our only hope of returning to a peaceful life rests in you. Please, you must find those boys and bring them home to their family. |"
+	    1, 5, TSFX_HEALER4 },
+	{ "Horazon was insane. Thereare forces with which one does not interfere. It would not surprise me if you found only the charred remains of this damned fool. |"

```

Comment:
```suggestion
	{ "Horazon was insane. There are forces with which one does not interfere. It would not surprise me if you found only the charred remains of this damned fool. |"

```